vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -544,6 +544,10 @@ public Element getElement() {
          * <p>
          * <strong>Note:</strong> calling this method automatically sets the
          * column as sortable with {@link #setSortable(boolean)}.
+         * <p>
+         * <strong>Note:</strong> Comparator is not serializable. If you need
+         * to write serializable implementation, use inlined class of 
```

Comment:
Please remove trailing whitespace in this line.

Reply:
Done